### DISCUSSION TITLE
kde blur

### DISCUSSION THREAD

**Discussion by ghost** [wezterm/wezterm#5829] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

kwin

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

is adding kde blur planned at all? applications e.g. alacritty offer a config option that can request kwin to blur their background

i saw https://github.com/wez/wezterm/issues/1614 but it seems like implementing this isn't at all a priority

**Comment by wez** [comment]
as that issue indicates, I don't have plans for this myself, but will accept a PR
